base/gfspapp/sign_server.go

```diff
@@ -41,6 +41,11 @@ func (g *GfSpBaseApp) GfSpSign(ctx context.Context, req *gfspserver.GfSpSignRequ
 		if err != nil {
 			log.CtxErrorw(ctx, "failed to seal object", "error", err)
 		}
+	case *gfspserver.GfSpSignRequest_RejectObjectInfo:
+		err = g.signer.RejectUnSealObject(ctx, t.RejectObjectInfo)
```

Comment:
fix `RejectUnSealObject` as `RejectUnsealedObject` may be better.

Reply:
keep same to greenfield

Reply:
greenfield used `NewMsgRejectUnsealedObject`